exporters/trace/README.md

```diff
@@ -1 +1,51 @@
 # Cloud Trace Exporter for OpenTelemetry
+
+  OpenTelemetry Google Cloud Trace Exporter allows the user to send collected traces to Google Cloud. 
+  
+  <a href="https://cloud.google.com/trace">Google Cloud Trace</a> is a distributed tracing backend system. It helps developers to gather timing data needed to troubleshoot latency problems in microservice & monolithic architectures. It manages both the collection and lookup of gathered trace data.
+
+## Setup
+  Google Cloud Trace is a managed service provided by Google Cloud Platform.
+
+## Installation
+  Not currently ready as a package to be imported. To use presently, one would have to clone this GitHub repo.
+## Usage
+  If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instructions in Authentication.  
+    
+  The TraceExporter constructor takes in three parameters: `projectId: String` for your GCP project ID, `traceServiceClient: TraceServiceClient` for the trace service client to eventually batch write spans, and `fixedAttributes: Map<String, AttributeValue>` for the fixed attributes of a span.  
+  So, we need to import the following: 
+  ```java
+  import com.google.cloud.trace.v2.TraceServiceClient;
+  import com.google.devtools.cloudtrace.v2.AttributeValue;
+  import com.google.devtools.cloudtrace.v2.ProjectName;
+  import com.google.devtools.cloudtrace.v2.Span;
+  ```
+  Declare and initialize the variables that will be used in the constructor parameters.  
+  Then, we can create a TraceExporter with, for example:
+  ```java
+  TraceExporter javaTraceExporter = new TraceExporter(projectId, traceServiceClient, fixedAttributes);
+  ```
+  Start tracing and collecting SpanData.  
+  Spans can be created by importing and using global `opentelemetry-java` API packages, for example:  
+  ```java
+  String operationName = "receive"; //or whatever other operation
+  Span span = this.tracer.spanBuilder(operationName).startSpan();
+  ```
+  To export these spans, configure the trace provider with the exporter:
```

Comment:
This section should be right after creation of the exporter, I think.
Otherwise there is an impression that one could start creating spans before registering the exporter.
You can just merge 2 sections together, something like this:

---

Declare and initialize the variables that will be used in the constructor parameters.  
Then, we can create and register TraceExporter, for example:

```java
TraceExporter javaTraceExporter = new TraceExporter(projectId, traceServiceClient, fixedAttributes);
OpenTelemetrySdk.getTracerProvider().addSpanProcessor(
  SimpleSpanProcessor.newBuilder(javaTraceExporter).build()
);
  ```

---

followed by "Start tracing and collecting SpanData..."

Reply:
Awesome, that clears up some confusion on my end too!